CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 - Pending via #211: Upgrade to GLFW 3.3
+- Add support for glfw's DisableCursor
+  - Closes https://github.com/faiface/pixel/issues/213
+
+## [v0.9.0] - 2020-05-02
+- Added feature from https://github.com/faiface/pixel/pull/219
```

Comment:
Looks alright, assuming #219 is part of the `v0.9.0` tag and #213 was merged afterwards :)

Reply:
Yes, thats the case :)